src/Util/Asserts.h

```diff
@@ -0,0 +1,30 @@
+//
+//  Asserts.h
+//
+//  Copyright (c) 2016 Open Whisper Systems. All rights reserved.
+//
+
```

Comment:
> I'm a fan of small, lean .pchs but this kind of asserts header is imo a strong candidate for inclusion, since ideally I'd like to use asserts in nearly every source file.

blergh. Not your fault, missed that line. Yes, this makes sense for the .pch.